control

```diff
@@ -8,9 +8,9 @@ Description: Adds custom devices to VeriStand.
 XB-LanguageSupport: en
-XB-AlwaysRemoves: ni-scan-engine-veristand-2019-support, ni-scan-engine-veristand-2020-support, ni-synchronization-veristand-2019-support, ni-synchronization-veristand-2020-support, ni-engine-simulation-veristand-2019-support, ni-engine-simulation-veristand-2020-support, ni-slsc12201-veristand-2019-support, ni-slsc12201-veristand-2020-support, ni-slsc-eds-veristand-2019-support, ni-slsc-eds-veristand-2020-support, ni-routing-and-faulting-veristand-2019-support, ni-routing-and-faulting-veristand-2020-support, ni-slsc-switch-veristand-2019-support, ni-slsc-switch-veristand-2020-support, ni-embedded-data-logger-veristand-2020-support, ni-routing-and-faulting-veristand-2019-labview-support, ni-routing-and-faulting-veristand-2020-labview-support, ni-slsc-switch-veristand-2019-labview-support, ni-slsc-switch-veristand-2020-labview-support, ni-routing-and-faulting-veristand-2019-labview-examples, ni-routing-and-faulting-veristand-2020-labview-examples, ni-instrument-addon-veristand-2019-support, ni-instrument-addon-veristand-2020-support,
+XB-AlwaysRemoves: ni-scan-engine-veristand-2019-support, ni-scan-engine-veristand-2020-support, ni-synchronization-veristand-2019-support, ni-synchronization-veristand-2020-support, ni-engine-simulation-veristand-2019-support, ni-engine-simulation-veristand-2020-support, ni-slsc12201-veristand-2019-support, ni-slsc12201-veristand-2020-support, ni-slsc-eds-veristand-2019-support, ni-slsc-eds-veristand-2020-support, ni-routing-and-faulting-veristand-2019-support, ni-routing-and-faulting-veristand-2020-support, ni-slsc-switch-veristand-2019-support, ni-slsc-switch-veristand-2020-support, ni-embedded-data-logger-veristand-2020-support, ni-routing-and-faulting-veristand-2019-labview-support, ni-routing-and-faulting-veristand-2020-labview-support, ni-slsc-switch-veristand-2019-labview-support, ni-slsc-switch-veristand-2020-labview-support, ni-routing-and-faulting-veristand-2019-labview-examples, ni-routing-and-faulting-veristand-2020-labview-examples, ni-instrument-addon-veristand-2019-support, ni-instrument-addon-veristand-2020-support, ni-fpga-addon-2019-support (>=20.2.0), ni-fpga-addon-2020-support (>=20.2.0)
```

Comment:
The versions should not be listed here. See other packages in this line for an example.

Reply:
Updated as suggested!